pkg/kubelet/dockertools/manager.go

```diff
@@ -64,6 +64,8 @@ const (
 	// we want to able to consider SRV lookup names like _dns._udp.kube-dns.default.svc to be considered relative.
 	// hence, setting ndots to be 5.
 	ndotsDNSOption = "options ndots:5\n"
+
+	containerNetworkEnvKey = "NETWORK"
```

Comment:
If we ever envision updates to network configurations, an ENV variable is not the best way to expose network information. Has the methodology for introspecting network config been discussed already with the community? 
@thockin @bgrant0607